packages/web-components/src/components/cbp-toast/cbp-toast.stories.tsx

```diff
@@ -43,12 +44,13 @@ export default {
   
   Toast.args = {
     open: true,
-    icon: 'user',
+    icon: `<cbp-icon size='2rem' name='user'/>`,
```

Comment:
revert this per previous comment. Just pass the name, render the icon in the story.